src/generateNumber.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+function generateNumber() {
+  const digits = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
+  const shuffled = digits.sort(() => Math.random() - 0.5);
```

Comment:
That's a cool way to do this, but not efficient nor truly random.

`.sort` is a [sorting algorithm](https://blog.shovonhasan.com/time-space-complexity-of-array-sort-in-v8/), and it takes many more comparisons than the number of digits in your array. Checkout the comments for this [stackOverflow answer](https://stackoverflow.com/a/18650169).